pkg/gadgets/common/mntns_filter.h

```diff
@@ -23,7 +26,7 @@ static __always_inline bool gadget_should_discard_mntns_id(__u64 mntns_id) {
 }
 
 // gadget_get_mntns_id returns the mntns_id of the current task.
-static __always_inline __u64 gadget_get_mntns_id() {
+static __always_inline mnt_ns_id_t gadget_get_mntns_id() {
```

Comment:
Should the same change be applied to gadget_should_discard_mntns_id()?

Reply:
see comment above.